src/quo/components/list_items/token_network/component_spec.cljs

```diff
@@ -0,0 +1,71 @@
+(ns quo.components.list-items.token-network.component-spec
+  (:require [quo.components.list-items.token-network.style :as style]
+            [quo.components.list-items.token-network.view :as token-network]
+            [quo.foundations.resources :as resources]
+            [test-helpers.component :as h]))
+
+(h/describe "List items: Token network"
+  (h/test "default state"
+    (h/render [token-network/view
+               {:token       :snt
+                :networks    [(resources/get-network :ethereum)]
+                :token-value "100.00 SNT"
+                :fiat-value  "€0.00"}])
+    (h/is-truthy (h/get-by-text "Status")))
+
+  (h/test "default state explicit"
+    (h/render [token-network/view
+               {:token       :snt
+                :state       :default
+                :networks    [(resources/get-network :ethereum)]
+                :token-value "100.00 SNT"
+                :fiat-value  "€0.00"}])
+    (h/is-truthy (h/get-by-text "Status")))
+
+  (h/test "Pressed state"
+    (let [state :pressed]
+      (h/render [token-network/view
+                 {:token       :snt
+                  :state       :default
+                  :networks    [(resources/get-network :ethereum)]
+                  :token-value "100.00 SNT"
+                  :fiat-value  "€0.00"}])
+      (h/fire-event :on-press-in (h/get-by-label-text :container))
+      (h/wait-for #(h/has-style (h/query-by-label-text :container)
+                                {:backgroundColor (:background-color
+                                                   (style/container state :blue :light))}))))
```

Comment:
I'd prefer to test this with the color value, could be `(colors/resolve-color :blue :light 5)`

Reply:
I think it makes more clear what is the expected value while reading the test. Otherwise, I would need to go to the style function, and check the expected value. Also if the style function changes because of some design requirement, I it may be the case that the test doesn't fail, while it probably should fail so the test can be adjusted accordingly to the new expected value.